browser/src/control/Control.NotebookbarWriter.js

```diff
@@ -138,7 +138,7 @@ L.Control.NotebookbarWriter = L.Control.Notebookbar.extend({
 		var hasShare = this._map['wopi'].EnableShare;
 		var hideDownload = this._map['wopi'].HideExportOption;
 		var hasGroupedSaveAs = window.prefs.get('saveAsMode') === 'group';
-		var hasRunMacro = !(window.enableMacrosExecution  === 'false');
+		var hasRunMacro = window.enableMacrosExecution;
```

Comment:
if the value is a string, then if we will pass string 'false' it will be used as boolean true if we do conditions like:
`if(enableMacrosExecution)`